terraform/fac.gov.tf

```diff
@@ -6,13 +6,29 @@ resource "aws_route53_zone" "fac_gov_zone" {
   }
 }
 
-#resource "aws_route53_record" "fac_gov__fac_gov_cname" {
-#  zone_id = aws_route53_zone.fac_gov_zone.zone_id
-#  name    = "fac.gov."
-#  type    = "CNAME"
-#  ttl     = 60
-#  records = ["fac.gov.external-domains-production.cloud.gov."]
-#}
+resource "aws_route53_record" "fac_gov_apex" {
+  zone_id = aws_route53_zone.fac_gov_zone.zone_id
+  name    = "fac.gov."
+  type    = "A"
+
+  alias {
+    name                   = "fac.gov.external-domains-production.cloud.gov."
```

Comment:
This should be the cloudfront URI not the subdomain of ____.cloud.gov.

Eg. "xxxxxxxxxxxxx.cloudfront.net."